temfpy/integration.py

```diff
@@ -123,9 +123,9 @@ def corner_peak(x, a):
     -----
     This function was proposed by Alan Genz in [G1984]_. It can be
     integrated analytically quickly with high precision. Evaluated at two

```

Comment:
'integrated analytically quickly' is an awkward type of phrasing. Unless there is a specific reason you would like to include the word 'analytically,' the easiest way to fix the sentence would be to remove it and just say 'integrated quickly'. Alternatively, you could say 'Analytically, it can be integrated quickly...' or 'It can be analytically integrated quickly...' Please let me know what you would prefer or if you have another suggestion that I can incorporate into the text.